src/Components/Browser.JS/jest.config.js

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+module.exports = {
+  globals: {
+    "ts-jest": {
+      "tsConfig": "./tests/tsconfig.json",
+      "babeConfig": true,
+      "diagnostics": true
+    }
+  },
+  preset: 'ts-jest',
+  testEnvironment: 'jsdom'
+};
```

Comment:
What causes the Jest tests to run in CI, and how does a developer run them manually locally? Is this tied into our .NET build system automatically by the AspNetCore build SDK?

Reply:
`.\build -test` just works, the build system knows about `npmproj` files that have `IsTestProject` set. Locally running `dotnet test` or `yarn test` works

Reply:
Wrt the debugger, I used a vscode extension called `jest`. I could check in the launch.json (I don't think other projects in the repo do this?), but it the auto-generated boiler plate.